de-de/uebersetzen.md

```diff
@@ -0,0 +1,66 @@
+
+### Mod Menu Helper (Mod-Menü-Helfer) übersetzen
+
+*Fabulously Optimized* beinhaltet ein Ressourcenpaket namens "Mod Menu Helper" das kurz beschreibt, welche Mods was macht. Standardmäßig ist es in englisch, aber ich bin offen, neue Übersetzungen hinzuzufügen. Folge also gerne diesem Tutorial:
+
+1. Nachdem du FO gestartet hast, gehe zum Ressourcenpakete-Menü und klicke *Open Pack Folder*
+2. Extrahiere *Mod Menu Helper.zip* in einen beliebigen Ordner, dann gehe in diesen Ordner → `assets` → `minecraft` → `lang`
+3. Öffne die Datei `en_us.json` und kopiere ihren Inhalt
+4. Erstelle eine neue Datei in dem Dateinamensformat `locale_code.json`, z.B. `et_ee.json` oder `de_de.json`. Schaue in [diese Tabelle](https://minecraft.fandom.com/wiki/Language#Languages) und suche den `in-game` Code, den du brauchst.
+5. Füge den Text von Schritt 3 hier ein und übersetze die Angaben rechts **nach** dem Doppelpunkt:
+
+```
+"coolmod.config.title": "Example Text",
+```
+**↓** Kann übersetzt werden zu **↓**
+```
+"coolmod.config.title": "Beispieltext",
+```
+
+1. Halte dich kurz und knapp.
+   * Ändere keine Farbcodes (§), den Stift-Emoji 🖉 oder "newlines" (Zeilenumbrüche, `\n`)
+2. Teste deine Übersetzungen im Spiel, indem du auf `Options...` → `Resource Packs...`, die "Mod Menu Helper.zip" deaktivierst und stattdessen "Mod Menu Helper" aktivierst und danach in das `Mods`-Menü schaust.
+   * Die Zeilen **müssen** bei einem maximierten Vollbildschirm-Fenster bei der *Full HD*-Auflösung (1920×1080) mit der Standardschrift (oder der Unicode Schrift, falls deine Sprache sie automatisch nutzt) vollständig sichtbar sein.
+   * Du kannst `F3+J` nutzen, um die Sprachdateien neuzuladen.
+3. Falls du mit deiner Übersetzung zufrieden bist, kopiere den Inhalt deiner Datei.
+4. [Klick hier](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/Mod%20Menu%20Helper/assets/modmenu/lang/), dann `Add file` → `Create new file`. Du brauchst dafür ein GitHub-Konto.
+5. Dies wird das Projekt "forken" und es wird sich ein neues Dateifenster öffnen. Nenne die Datei in dem Dateiformat von dem vorherigen Tutorial oben bei Schritt 4 und füge den kopierten Inhalt mit deiner neuen **Übersetzung** ein.
+> **Tipp:** Du kannst auch einfach über `Add File` → `Create new file` eine Datei hochladen 
+6. `Commit`e die Datei und eröffne einen `Pull Request`. Wenn deine Übersetzung perfekt in die Zeilen passt, dann werde ich sie sicherlich akzeptieren.
+7. Danke für einen Beitrag! Die Übersetzung wird wahrscheinlich in der nächsten Version des Modpacks veröffentlicht. 
+
+### Mods übersetzen
+
+1. [Schau dir die Liste der Mods an](https://github.com/Madis0/fabulously-optimized#included-mods)
+2. Finde auf jeder Website den entsprechenden `Issues` oder `Source` Link, siehe die Leiste oben.
```

Comment:
Isn't GitHub available in German? Maybe translate the links too :stuck_out_tongue:

Reply:
I think only the homepage and landing pages are translated. That's it.
![image](https://user-images.githubusercontent.com/67185896/163031345-3a2b2f55-ee6b-4a37-9f40-02044ac02498.png)

Reply:
I think for all the repos & users, GitHub just checks the user's browser/OS language.